.github/workflows/e2e-tests.yml

```diff
@@ -19,10 +24,18 @@ jobs:
         working-directory: ./e2e-tests
```

Comment:
Ikke fått med meg helt at dette ventet på meg 😳 

Egentlig ikke en del av denne endringen, men tidligere når workflowen først ble opprettet. Burde egentlig bruke samme cache som de andre workflows, og yarn for å være konsekvent med lock-fil. Se f.eks:

https://github.com/AtB-AS/webshop/blob/master/.github/workflows/deploy-prod.yml#L12-L21

Reply:
Lagt opp til å benytte yarn med yarn.lock nå (siste commit på tor/e2e-tester).

Men her blir jeg usikker på hvordan samme cache skal benyttes siden e2e-testene har node_modules inne i e2e-tests. Skal jeg bare re-bruke cache-steget fra referert yml, men endre path - eller vil "**/node_modules" også plukke opp den inne i e2e-tests?

Reply:
Jeg tror den vil bruke samme cache da ja